block-styles.css

```diff
@@ -0,0 +1,20 @@
+.wp-block-code textarea {
+	background: transparent;
+}
+
+.code-block-bg {
+	left: 0;
+	position: absolute;
+	top: .8em;
+	width: 100%;
+	z-index: -1;
```

Comment:
Note that opposed to positioning `.code-block-bg` it behind the `textarea`, you could go ahead and position it on top but give it `pointer-events:none`. This will prevent it from interfering with user clicks, while also avoid needing to make the `textarea` transparent.

Reply:
Great idea! Implemented